utils/bin/katello-disconnected

```diff
@@ -16,6 +16,12 @@
 # workaround for: https://github.com/grosser/fast_gettext/issues/32
 Encoding.default_external='UTF-8' if RUBY_VERSION >= '1.9'
 
+# mad h4x0rz for BZ 1241343
+# solution reference: http://lists.openshift.redhat.com/openshift-archives/dev/2014-July/msg00193.html
+ENV['LD_LIBRARY_PATH'] += ':/opt/rh/v8314/root/usr/lib64'
+require 'action_view'
+require 'v8'
+
```

Comment:
minor beef, any reason these requires aren't next to the rest of the listand sit above the APP_NAME declaration?